cirq-google/cirq_google/arg_func_langs.py

```diff
@@ -35,11 +33,14 @@
     # None means any. Is used when inferring the language during serialization.
     None: frozenset({'add', 'mul', 'pow'}),
 }
+MOST_PERMISSIVE_LANGUAGE = 'exp'
 
 SUPPORTED_SYMPY_OPS = (sympy.Symbol, sympy.Add, sympy.Mul, sympy.Pow)
 
 # Argument types for gates.
 ARG_LIKE = Union[int, float, List[bool], str, sympy.Symbol, sympy.Add, sympy.Mul]
+FLOAT_ARG_LIKE = Union[float, sympy.Symbol, sympy.Add, sympy.Mul]
+FLOAT_LIKE = (float, int, sympy.Integer, sympy.Float, sympy.Rational, sympy.NumberSymbol)
```

Comment:
nit: maybe name this differently to distinguish it from the `Union`s above, e.g. `FLOAT_LIKE_TYPES`. Might also be nice to add a comment to indicate why a tuple is needed here, instead of a union.

Reply:
Done.